code/dan/python/lab06.py

```diff
@@ -0,0 +1,88 @@
+#Version 1 & 2
+
+
+# Dictionary "Number of matches", "Jackpot Amount"
+winnings_dict= {
+                0:0,
+                1:4,
+                2: 7,
+                3 : 100,
+                4: 50_000,
+                5 : 1_000_000,
+                6 : 25_000_000,
+}
+import random
+
+#empty lists to append
+ticket = []
+winning_ticket = []
+balance = 0
+expense = 0
+
+# takes in a list adds 6 random numbers to list
+def pick6(list):
+    """
+    return a list of 6 random numbers
+    """
+    for x in range(6):
+      x = random.randint(1,99)
+      list.append(x)
+    return list 
+ 
+# print function passing in empty list variables  
+print(pick6(ticket))
+print(pick6(winning_ticket))
+
+# function to check numbers in ticket & winning ticekt match in correct index
+
+def num_matches(ticket, winning_ticket):
+    matches = 0
+    #global matches # global used to return matches outside of function 
+    for x in range (len(ticket)):
+            if ticket[x] == winning_ticket[x]:
+                matches += 1
+            return matches
```

Comment:
@asddataking there is a bit of a problem with this function here.  `return matches` is inside the for loop.  It will return after checking the first value every time.  So there will be 1 match if the first number matches, and 0 matches in every other case.